cms/utils/decorators.py

```diff
@@ -20,6 +20,8 @@ def inner(request, *args, **kwargs):
         return func(request, *args, **kwargs)
     inner.__module__ = func.__module__
     inner.__doc__ = func.__doc__
+    if hasattr(func, "view_class"):
```

Comment:
@Will-Hoey I'd prefer a conditional based on the Django version to keep the semantics and be able to remove the conditional completely once Django 3.2 is ancient history.